From bb429eb1f018bb8d996fb6faa413af5a8025b194 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Petr=20P=C3=ADsa=C5=99?= <ppisar@redhat.com>
Date: Fri, 20 Jan 2023 11:28:35 +0100
Subject: [PATCH] Do not ignore Swig failures
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

If private.c and private.pm files need to be regenerated by Swig,
failures of swig command were silently ignored.

This patch fixes it by failing Makefile on swig error.

Signed-off-by: Petr Písař <ppisar@redhat.com>
---
 Makefile.PL | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.PL b/Makefile.PL
index a00d23b..f571f06 100644
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -102,7 +102,7 @@ sub MY::postamble {
 pm_to_blib: lib/Text/Bidi/private.pm
 
 private.c lib/Text/Bidi/private.pm: swig/fribidi.i
-	-/usr/bin/swig -perl -Wall -I/usr/include $cflags -outdir lib/Text/Bidi/ -o private.c \$<
+	/usr/bin/swig -perl -Wall -I/usr/include $cflags -outdir lib/Text/Bidi/ -o private.c \$<
  
 swig-clean: clean
 	-rm private.c lib/Text/Bidi/private.pm
-- 
2.39.1

